pay-api/src/pay_api/services/payment.py

```diff
@@ -352,21 +352,11 @@ def create_payment_report_details(cls, purchases: Tuple, data: Dict):  # pylint:
             data = {'items': []}
 
         for invoice_dao in purchases:
-            included_fields = ['id', 'corp_type_code', 'created_on', 'invoice_status_code', 'total', 'paid',
-                               'service_fees', 'references', 'folio_number', 'created_name', 'payment_method_code',
-                               'details', 'payment_account', 'business_identifier', 'created_by']
-            invoice_schema = InvoiceSchema(only=included_fields)
-            invoice = invoice_schema.dump(invoice_dao)
-            invoice['line_items'] = []
-            for payment_line_item in invoice_dao.payment_line_items:
-                # All we need for CSV and PDF and search.
-                line_item_schema = PaymentLineItemSchema(only=('gst', 'description', 'pst'))
-                line_item_dict = line_item_schema.dump(payment_line_item)
-                line_item_dict['filing_type_code'] = payment_line_item.fee_schedule.filing_type_code
-                invoice['line_items'].append(line_item_dict)
-                invoice['product'] = payment_line_item.fee_schedule.corp_type.product
+            invoice = InvoiceSchemaSearchModel.from_orm(invoice_dao).dict()
             if len(invoice_dao.references) > 0:
                 invoice['invoice_number'] = invoice_dao.references[0].invoice_number
```

Comment:
We should probably remove these, or there might be a better way to do it in pydantic.